biogeochem/EDPatchDynamicsMod.F90

```diff
@@ -2255,6 +2383,19 @@ subroutine fuse_patches( csite, bc_in )
        enddo !do while nopatches>maxPatchesPerSite
 
     end do  ! i_disttype loop
+
+    currentPatch => currentSite%youngest_patch
+    do while(associated(currentPatch))
+
+       if (currentPatch%anthro_disturbance_label .eq. primaryforest) then
+          primary_land_fraction_afterfusion = primary_land_fraction_afterfusion + &
+               currentPatch%area * AREA_INV
+       endif
+
+       currentPatch => currentPatch%older
+    enddo
+
+    currentSite%primary_land_patchfusion_error = primary_land_fraction_afterfusion - primary_land_fraction_beforefusion
```

Comment:
what happens to this? Does it get checked somewhere?

Reply:
yes, it becomes a history variable.  its possible that we don't need it, I had only added it to try to test a hypothesis about what was going wrong, and found that the values of it were very small.